analyzer/emerald/incoming_test.go

```diff
@@ -0,0 +1,28 @@
+package emerald
+
+import (
+	"context"
+	"encoding/hex"
+	"testing"
+
+	"github.com/oasisprotocol/oasis-core/go/runtime/client/api"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
+	"github.com/stretchr/testify/require"
+
+	"github.com/oasisprotocol/oasis-indexer/cmd/common"
+	"github.com/oasisprotocol/oasis-indexer/storage/oasis"
+)
+
+func TestTokenDataERC20(t *testing.T) {
+	// todo: can't keep this
+	ctx := context.Background()
+	cf, err := oasis.NewClientFactory(ctx, config.DefaultNetworks.All["mainnet"])
```

Comment:
let's delete this test for now.
after the mvp, let's come up with a way to deploy a token in our test fixture and reference that

Reply:
right, not offline. external dependencies

I've only used this to develop the reader, so that I could invoke the new code easily without actually scanning blocks

ok let's keep this. devs feel free to remove it if mainnet gets annihilated

Reply:
changed commit message. filed #250